src/utilities/date.js

```diff
@@ -0,0 +1,11 @@
+export const countdown = (startDate, endDate) => {
+  let dayDifference = endDate.getDay() - startDate.getDay();
+  let hourDifference = endDate.getHours() - startDate.getHours();
+  dayDifference = hourDifference < 0 ? dayDifference - 1 : dayDifference;
+  hourDifference = hourDifference < 0 ? 24 + hourDifference : hourDifference;
+
+  if (dayDifference < 0) {
+    return "This Auction Has Ended";
```

Comment:
This presentation messaging is pretty specific, while the function seems like it should be pretty generic.

Reply:
We'll break these up into separate functions